gnome-shell/gnome-shell-sync.sh

```diff
@@ -53,9 +53,44 @@ root=${data}/theme
 
 [ ! -d ${_destination} ] && echo ${_destination} folder does not exists && exit 1
 [ ! -d ${_destination}/gnome-shell-sass ] && mkdir ${_destination}/gnome-shell-sass
+[ ! -d ${_destination}/gnome-shell-sass/widgets] && mkdir ${_destination}/gnome-shell-sass/widgets
 [ ! -d ${_destination}/data ] && mkdir ${_destination}/data
 
 files=(
```

Comment:
Seems this script comes from Ubuntu Yaru: https://github.com/ubuntu/yaru/blob/master/gnome-shell/gnome-shell-sync.sh

I wonder why they wrote it to explicitly list only these files...

Reply:
My guess is that they manually tracked changes in 3.36 and 3.38, and that script is somewhat vestigial in Yaru at this point. It hasn't been updated since March.

Reply:
Probably something like that.